package.json

```
@@ -31,6 +31,13 @@
   "main": "./dist/lib/server-impl.js",
   "scripts": {
     "start": "TZ=UTC node ./dist/server.js",
+    "build:backend": "copyfiles -u 1 src/mailtemplates/**/*.mustache dist/ && tsc --pretty --strictNullChecks false",
+    "build:frontend": "cd ./frontend && yarn run build",
```

I like this because it doesn't require a change directory. Although it might seem just a preference it may matter when you combine `build:frontend` with something else (but not 100% sure)
```suggestion
    "build:frontend": "yarn --cwd ./frontend run build",
```